editor/editor_inspector.cpp

```diff
@@ -1701,9 +1701,12 @@ void EditorInspector::update_property(const String &p_prop) {
 
 void EditorInspector::_clear() {
 
-	while (main_vbox->get_child_count()) {
-		memdelete(main_vbox->get_child(0));
+	for (int i = main_vbox->get_child_count(); i-- > 0;) {
```

Comment:
Please use `i > 0; i--` instead of this shortcut, it's what we use everywhere else.

Reply:
Updated to `int i = main_vbox->get_child_count() - 1; i >= 0; i--` to match style

(For reference, the previous version isn't a shortcut. It's the form of the loop that also works for an unsigned type like size_t)

Reply:
Ah thanks, that makes sense and it's good to know :)